src/org/opensolaris/opengrok/history/FileHistoryCache.java

```diff
@@ -64,10 +64,11 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(FileHistoryCache.class);
 
     private final Object lock = new Object();
-    private String historyCacheDirName = "historycache";
-    private String latestRevFileName = "OpenGroklatestRev";
+    private final String historyCacheDirName = "historycache";
+    private final String latestRevFileName = "OpenGroklatestRev";
     private boolean historyIndexDone = false;
-    
+    public final static String historyDirName = "historycache";
```

Comment:
I wanted to use the dirname either in the tests or elsewhere (which is bad idea really as everything from within `IndexDatabase` et al. should go through `HistoryGuru` and not directly to `FileHistoryCache`) so made a public duplicate. Removed the public object.